### PR TITLE
Repo Activity: count new issues that were closed

### PR DESCRIPTION
**Pull request by tmnvanderberg**
I'm new to go and contributing to gitea, your guidance is much appreciated.

This is meant to solve https://github.com/go-gitea/gitea/issues/13309

Previously, closed issues would not be shown under new issues in the
activity tab, even if they were newly created.

changes:
* Split out newlyCreatedIssues from issuesForActivityStatement to count
  both currently open and closed issues.
* Use a seperate function to count active issues to prevent
double-counting issues after the above change.

Result is that new issues that have been closed are shown both under "new" and "closed".

### CODE REVIEW

File: models/activities/repo_activity.go [hunk 1]
```diff
@@ -172,7 +173,7 @@ func (stats *ActivityStats) MergedPRPerc() int {
 
 // ActiveIssueCount returns total active issue count
```
Comment (lunny) [inline models/activities/repo_activity.go:174]: The comment needs to be changed.

File: models/activities/repo_activity.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ type ActivityStats struct {
 	OpenedPRAuthorCount         int64
 	MergedPRs                   issues_model.PullRequestList
 	MergedPRAuthorCount         int64
+	ActiveIssues                issues_model.IssueList
```
Comment (lunny) [inline models/activities/repo_activity.go:37]: Where is the field being used?

Comment (tmnvanderberg) [reply models/activities/repo_activity.go:37]: Lines 176, 299, 300

File: models/activities/repo_activity.go (hunk 1 above)
Comment (tmnvanderberg) [reply models/activities/repo_activity.go:174]: Not sure I follow. We are returning the total number of active issues (actually, before we were not).

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
last call for merge @go-gitea/technical-oversight-committee

File: models/activities/repo_activity.go [hunk 3]
```diff
@@ -317,6 +326,23 @@ func (stats *ActivityStats) FillUnresolvedIssues(ctx context.Context, repoID int
...
+	sess := db.GetEngine(ctx).Where("issue.repo_id = ?", repoID).
+		And("issue.is_pull = ?", false).                // Retain the is_pull check to exclude pull requests
+		And("issue.created_unix >= ?", fromTime.Unix()) // Include all issues created after fromTime
+
+	return sess
+}
+
+func activeIssues(ctx context.Context, repoID int64, fromTime time.Time) *xorm.Session {
+	sess := db.GetEngine(ctx).Where("issue.repo_id = ?", repoID).
+		And("issue.is_pull = ?", false).
```
Comment (delvh) [inline models/activities/repo_activity.go:339]: I am a bit confused, in the issue it explicitly states:
> For PRs, it is basically the same issue.

but they are excluded right now?
So do they have the same problem or not?

Apart from that, LGTM.

Comment (tmnvanderberg) [reply models/activities/repo_activity.go:339]: You're right, pull requests suffer from the same problem. So this fix is only for part of the problem affecting the issue counts.

However, that is unrelated to this code snippet which is counting issues, and hence should filter out pull requets.

 If this PR is approved I will be happy to submit a similar fix for pull requests.

**Review by delvh — APPROVED** [review]

**Pull request merged**